www/about.rst

```diff
@@ -38,25 +38,23 @@ Core Packages
 * pandas_, providing high-performance, easy to use data structures.
 * SymPy_, for symbolic mathematics and computer algebra.
 * IPython_, a rich interactive interface, letting you quickly process data and
-  test ideas. The **IPython notebook** works in your web browser, allowing you
-  to document your computation in an easily reproducible form.
-* nose_, a framework for testing Python code.
```

Comment:
I guess this is the controversial one.  Maybe mention nose and pytest where nose is still used but is gradually being phased out in favor of pytest?